docs/readthedocs/usersguide.md

```diff
@@ -108,7 +108,7 @@ Within SIMONA we use the PowerSystemDataModel (PSDM) for modeling the system com
 Before the data can be utilized for a simulation run, make sure to convert them to the PSDM.
 For more information on the PSDM visit the [docs](https://powersystemdatamodel.readthedocs.io/en/latest/index.html) and for an example of how the converted data looks like you can take a look at an example grid at ``./input/samples/vn_simona/fullGrid``.
 The example grids are provided as csv files. You can choose to use a different data source for your own grid.
-For more information on supported sources, check out the /config/config section.
+Supported data sources are: influxdb1x, csv, sql, couchbase.
```

Comment:
That's not true for the whole PSDM. For example, grid and asset inputs are currently only supported as csv. So, I'd still link the config section here :)
Furthermore, the according PSDM readthedocs pages, e.g. [this](https://powersystemdatamodel.readthedocs.io/en/latest/models/models.html#input) and [this](https://powersystemdatamodel.readthedocs.io/en/latest/io/basiciousage.html), could be linked here, or better in the config page.